lib/sass/script/functions.rb

```diff
@@ -2393,13 +2456,17 @@ def global_variable_exists(name)
     #
     # @overload function_exists($name)
     #   @param name [Sass::Script::Value::String] The name of the function to
-    #     check.
+    #     check or a callable.
```

Comment:
@nex3 The issue is that callables can refer to functions that don't exist. Initially my code raised an error if they tried to do that, but as I implemented it, the code really felt like matching the semantics of `call` (which can be used with functions that don't exist -- they become css function calls) made the most sense and the cleanest implementation. Once I made that change, it followed that `function-exists()` needed to accept a callable.  Let's discuss this in our weekly meeting.